plugins/processors/converter/converter.go

```diff
@@ -73,15 +76,11 @@ func (p *Converter) Description() string {
 	return "Convert values to another metric value type"
 }
 
-func (p *Converter) Apply(metrics ...telegraf.Metric) []telegraf.Metric {
-	if !p.initialized {
-		err := p.compile()
-		if err != nil {
-			logPrintf("initialization error: %v\n", err)
-			return metrics
-		}
-	}
+func (p *Converter) Init() error {
```

Comment:
is this a contract change that we need to worry about? previously Apply would init for you.

Reply:
The Init() function is optional on any plugin type, but will be called if defined.  It makes it a little bit easier to initialize a plugin without needing the manage it and improves the plugins ability to report errors with the configuration was setup improperly.